Formula/visp.rb

```diff
@@ -0,0 +1,65 @@
+class Visp < Formula
+  desc "Visual Servoing Platform library"
+  homepage "https://visp.inria.fr"
+  url "https://gforge.inria.fr/frs/download.php/latestfile/475/visp-3.1.0.tar.gz"
+  sha256 "2a1df8195b06f9a057bd4c7d987697be2fdcc9d169e8d550fcf68e5d7f129d96"
+
+  depends_on "cmake"     => :build
+  depends_on "eigen"
+  depends_on "gsl"
+  depends_on "jpeg"
+  depends_on "libdc1394"
+  depends_on "libpng"
+  depends_on "opencv"
+  depends_on "zbar"
+  depends_on :x11
+
+  def install
+
+    args = std_cmake_args + %w[
+      -DBUILD_DEMOS=OFF
+      -DBUILD_EXAMPLES=OFF
+      -DBUILD_TESTS=OFF
+      -DBUILD_TUTORIALS=OFF
+      -DUSE_DC1394=ON
+      -DUSE_EIGEN3=ON
+      -DUSE_GSL=ON
+      -DUSE_JPEG=ON
+      -DUSE_LAPACK=OFF
+      -DUSE_LIBUSB_1=OFF
+      -DUSE_OPENCV=ON
+      -DUSE_PCL=OFF
+      -DUSE_PNG=ON
+      -DUSE_PTHREAD=ON
+      -DUSE_PYLON=OFF
+      -DUSE_REALSENSE=OFF
+      -DUSE_REALSENSE2=OFF
+      -DUSE_X11=ON
+      -DUSE_XML2=ON
+      -DUSE_ZBAR=ON
+      -DUSE_ZLIB=OFF
+      -DWITH_LAPACK=OFF
+    ]
+
+    mkdir "build" do
+      system "cmake", "..", *args
```

Comment:
all of the arguments should be inlined since none are appended optionally

Reply:
You mean that I should remove all the `args` block
````
    args = std_cmake_args + %w[
      -DBUILD_DEMOS=OFF
      -DBUILD_EXAMPLES=OFF
      -DBUILD_TESTS=OFF
      -DBUILD_TUTORIALS=OFF
      -DUSE_DC1394=ON
      -DUSE_EIGEN3=ON
      -DUSE_GSL=ON
      -DUSE_JPEG=ON
      -DUSE_LAPACK=OFF
      -DUSE_LIBUSB_1=OFF
      -DUSE_OPENCV=ON
      -DUSE_PCL=OFF
      -DUSE_PNG=ON
      -DUSE_PTHREAD=ON
      -DUSE_PYLON=OFF
      -DUSE_REALSENSE=OFF
      -DUSE_REALSENSE2=OFF
      -DUSE_X11=OFF
      -DUSE_XML2=ON
      -DUSE_ZBAR=ON
      -DUSE_ZLIB=OFF
      -DWITH_LAPACK=OFF
    ]
````

and put instead 
````
system "cmake", ".", *std_cmake_args, 
      "-DBUILD_DEMOS=OFF", 
      "-DBUILD_EXAMPLES=OFF",
      "-DBUILD_TESTS=OFF",
      "-DBUILD_TUTORIALS=OFF",
      "-DUSE_DC1394=ON",
      "-DUSE_EIGEN3=ON",
      "-DUSE_GSL=ON",
      "-DUSE_JPEG=ON",
      "-DUSE_LAPACK=OFF",
      "-DUSE_LIBUSB_1=OFF",
      "-DUSE_OPENCV=ON",
      "-DUSE_PCL=OFF",
      "-DUSE_PNG=ON",
      "-DUSE_PTHREAD=ON",
      "-DUSE_PYLON=OFF",
      "-DUSE_REALSENSE=OFF",
      "-DUSE_REALSENSE2=OFF",
      "-DUSE_X11=OFF",
      "-DUSE_XML2=ON",
      "-DUSE_ZBAR=ON",
      "-DUSE_ZLIB=OFF",
      "-DWITH_LAPACK=OFF"
````

I'm right ?

Reply:
```
    system "cmake", ".", "-DBUILD_DEMOS=OFF",
                         "-DBUILD_EXAMPLES=OFF",
                         "-DBUILD_TESTS=OFF",
                         "-DBUILD_TUTORIALS=OFF",
                         "-DUSE_DC1394=ON",
                         "-DUSE_EIGEN3=ON",
                         "-DUSE_GSL=ON",
                         "-DUSE_JPEG=ON",
                         "-DUSE_LAPACK=OFF",
                         "-DUSE_LIBUSB_1=OFF",
                         "-DUSE_OPENCV=ON",
                         "-DUSE_PCL=OFF",
                         "-DUSE_PNG=ON",
                         "-DUSE_PTHREAD=ON",
                         "-DUSE_PYLON=OFF",
                         "-DUSE_REALSENSE=OFF",
                         "-DUSE_REALSENSE2=OFF",
                         "-DUSE_X11=OFF",
                         "-DUSE_XML2=ON",
                         "-DUSE_ZBAR=ON",
                         "-DUSE_ZLIB=OFF",
                         "-DWITH_LAPACK=OFF",
                         *std_cmake_args
```